src/mantarray_desktop_app/mc_comm.py

```diff
@@ -323,6 +332,39 @@ def _handle_incoming_data(self) -> None:
         else:
             raise UnrecognizedSerialCommModuleIdError(module_id)
 
+    def _process_main_module_comm(self, comm_from_instrument: bytes) -> None:
+        packet_body = comm_from_instrument[
+            SERIAL_COMM_ADDITIONAL_BYTES_INDEX:-SERIAL_COMM_CHECKSUM_LENGTH_BYTES
+        ]
+
+        packet_type = comm_from_instrument[SERIAL_COMM_PACKET_TYPE_INDEX]
```

Comment:
is there any need for error checking on the length of `comm_from_instrument`?

Reply:
There is, but it hasn't been determined yet what we should do if the length is less than the amount we tried to read. The way bytes are read from both the simulator and live board we will never get more bytes than we attempt to read. I put a TODO for this on line 307

Reply:
👍